front/src/modules/simulationResult/components/SpaceTimeChart/types.ts

```diff
@@ -1,6 +1,6 @@
 import { Chart, TrainsWithArrivalAndDepartureTimes } from 'reducers/osrdsimulation/types';
 
-export type DispatchUpdateChart = (chart: Chart) => void;
+export type DispatchUpdateChart = (chart: Chart | undefined) => void;
```

Comment:
See related comment in previous file
```suggestion
export type DispatchUpdateChart = (chart: Chart) => void;
```